_data/meltano/files/superset/meltano.yml

```diff
@@ -0,0 +1,8 @@
+name: superset
+namespace: superset
+label: Apache Superset
+description: A modern, enterprise-ready business intelligence web application.
+repo: https://github.com/meltano/files-superset
+pip_url: git+https://github.com/meltano/files-superset.git
+variant: meltano
+logo_url: /assets/logos/files/superset.png
```

Comment:
Not sure if this is needed if the name matches the expected pattern.

Reply:
Regarding logo_url, I don't know how the api generation script leverages this, but the goal is to have an fully qualified URL exposed in the API endpoint. Towards this end, I do prefer having the logo_url everywhere just so we're not relying on over-relying on convention.

If the API generation process is able to adapt and map correctly, I'd be okay with any of these three:

```
logo_url: superset.png                     # when no path part is provided, find the file in `/assets/logos/files/`
logo_url: /assets/logos/files/superset.png # abs url will be https://raw.githubusercontent.com/meltano/hub/main/assets/logos/files/airflow.png
logo_url: https://raw.githubusercontent.com/meltano/hub/main/assets/logos/files/airflow.png
```

Sidebar (and I'll open an issue for this): I don't think image storage should be brand-specific or logo-specific versus plugin_type-specific. I think we should consider `/assets/logos/` instead of `/assets/logos/files/`.

For instance, `tap-snowflake`, `target-snowflake`, `dbt-snowflake`, and `files-dbt-snowflake` might all use `https://raw.githubusercontent.com/meltano/hub/main/assets/logos/snowflake.png`. If we establish that logos are (by default) in `/assets/logos/`, then all we need here is `snowflake.png`. And if/when we need to, we can use slightly varying logo files like `dbt-snowflake.png` or `snowflake.jpg`, and this gives us control, flexibility, and berevity.

Reply:
https://github.com/meltano/hub/issues/545#issue-1260639744

Reply:
@aaronsteers yeah the API generation script does leverage it and adds the base url to it like `https://hub.meltano.com` or `http://localhost:4000` or similar, depending on where you're running it.

> Sidebar (and I'll open an issue for this): I don't think image storage should be brand-specific or logo-specific versus plugin_type-specific. I think we should consider /assets/logos/ instead of /assets/logos/files/.

Yeah I also agree we could definitely share the logos better, we can do that in a follow on though.